tests/framework/extensions/clusters/clusters.go

```diff
@@ -912,6 +912,96 @@ func CreateK3SRKE2Cluster(client *rancher.Client, rke2Cluster *apisV1.Cluster) (
 	return cluster, nil
 }
 
+// DeleteKE1Cluster is a "helper" functions that takes a rancher client, and the rke1 cluster config as parameters. This function
+// registers a delete cluster fuction with a wait.WatchWait to ensure the cluster is removed cleanly.
+func DeleteRKE1Cluster(client *rancher.Client, clusterID string) error {
+	cluster, err := client.Management.Cluster.ByID(clusterID)
+	if err != nil {
+		return err
+	}
+
+	adminClient, err := rancher.NewClient(client.RancherConfig.AdminToken, client.Session)
+	if err != nil {
+		return err
+	}
+
+	watchInterface, err := adminClient.GetManagementWatchInterface(management.ClusterType, metav1.ListOptions{
```

Comment:
This part should be in the verification. And the delete func should go in provisioning, or here. What do you think @slickwarren